sunpy/timeseries/timeseriesbase.py

```diff
@@ -68,25 +68,21 @@ class GenericTimeSeries:
     """
     A generic time series object.
 
+    Parameters
+    ----------
+    data : `~pandas.DataFrame`
+        A pandas DataFrame representing one or more fields as a function
+        of time.
+    meta : `~sunpy.timeseries.metadata.TimeSeriesMetaData`, optional
+        The metadata giving details about the time series data/instrument.
+
     Attributes
     ----------
+    data : `~pandas.DataFrame`
+        A pandas DataFrame representing one or more fields as a function
+        of time.
     meta : `~sunpy.timeseries.metadata.TimeSeriesMetaData`
         The metadata giving details about the time series data/instrument.
-    data : `~pandas.DataFrame`
-        An pandas DataFrame prepresenting one or more fields as a function of time.
-
```

Comment:
Do we need both Attributes and Parameters?

Reply:
Attributes are class attributes, and parameters are things passed to `__init__`, so they're different?

Reply:
Just that they seemed the same, can't we merge the two into one?

Reply:
yeah they need to be in both places.